lib/benchee/formatters/html.ex

```diff
@@ -194,30 +197,79 @@ defmodule Benchee.Formatters.HTML do
 
   ## Exmaples
 
-      iex> statistics = %{
-      ...>   "Job"   => %{average: 500.0},
-      ...>   "Other" => %{average: 200.0}
-      ...> }
-      iex> run_times = %{"Job" => [400, 600], "Other" => [150, 250]}
-      iex> Benchee.Formatters.HTML.merge_job_measurements(statistics, run_times)
+      iex> scenarios = [
+      ...>   %Benchee.Benchmark.Scenario{
+      ...>     job_name: "Job",
+      ...>     run_times: [400, 600],
+      ...>     run_time_statistics: %Benchee.Statistics{
+      ...>       average:       500.0,
+      ...>       ips:           2000.0,
+      ...>       std_dev:       20,
+      ...>       std_dev_ratio: 0.1,
+      ...>       std_dev_ips:   500,
+      ...>       median:        190.0,
+      ...>       sample_size:   3,
+      ...>       minimum:       190,
+      ...>       maximum:       210
+      ...>     }
+      ...>   },
+      ...>   %Benchee.Benchmark.Scenario{
+      ...>     job_name: "Other",
+      ...>     run_times: [150, 250],
+      ...>     run_time_statistics: %Benchee.Statistics{
+      ...>       average:       200.0,
+      ...>       ips:           5000.0,
+      ...>       std_dev:       20,
+      ...>       std_dev_ratio: 0.1,
+      ...>       std_dev_ips:   500,
+      ...>       median:        190.0,
+      ...>       sample_size:   3,
+      ...>       minimum:       190,
+      ...>       maximum:       210
+      ...>     }
+      ...>   }
+      ...> ]
+      iex> Benchee.Formatters.HTML.format_runtime_stats(scenarios)
       %{
         "Job" => %{
-          statistics: %{average: 500.0},
+          statistics: %Benchee.Statistics{
+            average:       500.0,
+            ips:           2000.0,
+            std_dev:       20,
+            std_dev_ratio: 0.1,
+            std_dev_ips:   500,
+            median:        190.0,
+            sample_size:   3,
+            minimum:       190,
+            maximum:       210
+          },
           run_times:  [400, 600]
         },
         "Other" => %{
-          statistics: %{average: 200.0},
+          statistics: %Benchee.Statistics{
+            average:       200.0,
+            ips:           5000.0,
+            std_dev:       20,
+            std_dev_ratio: 0.1,
+            std_dev_ips:   500,
+            median:        190.0,
+            sample_size:   3,
+            minimum:       190,
+            maximum:       210
+          },
           run_times: [150, 250]
         }
       }
   """
-  def merge_job_measurements(statistics, run_times) do
-    Map.merge(statistics, run_times, fn(_key, stats, times) ->
-      %{
-        statistics: stats,
-        run_times: times
-      }
-    end)
+  def format_runtime_stats(scenarios) do
```

Comment:
Looking at it again, I think `format_job_measurements` is more clear. At this point all the scenarios are for a single input, and we're grouping and formatting by job name, so I like having `job` in the function name here.